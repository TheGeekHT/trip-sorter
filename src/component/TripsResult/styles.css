.result {
    font-size: 20px;
    paddig: 8px 0;
    line-height: 1.5em;
}

.result p {
    margin-bottom: 25px;
}
.result p:last-child {
    margin-bottom: 0;
}

.result strong{
    white-space: nowrap;
}