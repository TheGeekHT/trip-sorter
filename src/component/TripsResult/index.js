// @flow

import React from 'react'
import { Panel } from 'react-bootstrap'

import './styles.css'

type Props = {
  children?: typeof React.PropTypes.node,
}

const TripsResult = ({ children }: Props) =>
  <Panel bsStyle="info" className="result">
    {children}
  </Panel>


export default TripsResult
