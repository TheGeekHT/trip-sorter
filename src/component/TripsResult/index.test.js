import React from 'react'
import renderer from 'react-test-renderer'
import TripsResult from './'

test('trips result should render without crashing', () => {
  const component = renderer.create(
    <TripsResult>
      Some test goes here
    </TripsResult>
  ).toJSON();
  expect(component).toMatchSnapshot();
});
