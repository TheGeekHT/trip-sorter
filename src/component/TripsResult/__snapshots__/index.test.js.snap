exports[`test trips result should render without crashing 1`] = `
<div
  className="result panel panel-info"
  id={undefined}>
  <div
    className="panel-body">
    Some test goes here
  </div>
</div>
`;
